assignments/4. operators/logical-operators.md

```diff
@@ -5,32 +5,32 @@
 * [ ] Logical AND operation
 
 ```js
-true  && true; //output
-true  && false;
-false && true;
-false && false;
-"foo" && "bar";
-"bar" && "foo";
-"foo" && "";
-""    && "foo";
-" "   && "John" && "" && false
-false && "Hey" && undefined
-"undefined" && false && 42
+true  && true; //output true
+true  && false;//output false
+false && true;// output false
+false && false;//output false
+"foo" && "bar";//output bar
+"bar" && "foo";//output foo
+"foo" && "";//output ""
+""    && "foo";// output ""
+" "   && "John" && "" && false//output ""
```

Comment:
`" "` first one is a string with space. That's a truthy value so the output will be `false`